source/languages/en/riak/community/product-advisories/appconfig-forward.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Forward Incompatibility of app.config
+project: riak
+version: 1.0.0+ 
+versions: false
+document: reference
+---
+
+Info | Value
+:----|:-----
+Date issued | April 15, 2015
+Product | Riak
+Affected Riak versions | 2.0.0+, 2.1.0+
+
+## Overview
+
+A forward incompatibility of app.config between 1.x and 2.x has been identified.
```

Comment:
After thinking about it some, I think we should just add the second `.x`, so `1.x` would become `1.x.x`. It's kinda weird, but so is our versioning system.